binary/grakn

```diff
@@ -71,7 +71,10 @@ elif [[ "$1" = "server" ]] || [[ "$1" = "version" ]]; then
         CLASSPATH="${GRAKN_HOME}/server/conf/:${GRAKN_HOME}/${LIB_COMMON}"
         # exec replaces current shell process with java so no commands after these ones will ever get executed
         if [ $DEBUG ]; then
-            CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/lib/dev/*"
+            case "$(uname -s)" in
+                Darwin) CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/lib/dev/*" ;;
+                *) CLASSPATH="${CLASSPATH}:${GRAKN_HOME}/server/lib/prod/*" ;;
```

Comment:
Let's add a warning: `WARNING: Debug mode is not available in this platform`. Silently disabling debug, means we're misleading the user.